cmake/tribits.cmake

```diff
@@ -300,7 +300,9 @@ FUNCTION(TRIBITS_ADD_EXECUTABLE EXE_NAME)
   ENDIF()
 ENDFUNCTION()
 
-ADD_CUSTOM_TARGET(check COMMAND ${CMAKE_CTEST_COMMAND} -VV -C ${CMAKE_CFG_INTDIR})
+IF(NOT TARGET check)
+  ADD_CUSTOM_TARGET(check COMMAND ${CMAKE_CTEST_COMMAND} -VV -C ${CMAKE_CFG_INTDIR})
```

Comment:
Could you please clarify why you needed to do this?  Thanks!

Reply:
Certainly. I have a number of CMake projects which I combine into one bundle using "add_subdirectory". It so happens that the target "check" was already defined previous to adding kokkos and in fact with a very similar command. This guard prevents it from being added twice.

Reply:
@wdeconinck Could you please separate that into a different pull request?  Sorry to be so picky :) .

Reply:
Okay, no problem. Stay tuned.